index.html

```diff
@@ -184,6 +184,50 @@ <h2 class="about-title">About Me</h2>
             <li class="skill">Gitlab</li>
         </ul>
     </section>
+    <footer>
+        <section>
+            <h2 class="titleFooter">
+                I'm always interested in hearing about 
+                new projects, so if you'd like to chat 
+                please get in touch.
+            </h2>
+            <form action="" class="form">
+                <input class="form-list" type="text" placeholder="Full Name" required maxlength="30">
+                <input class="form-list" type="email" placeholder="Email Address" required>
+                <textarea class="form-textarea" placeholder="Write me something..." maxlength="500" required></textarea>    
+                <button class="botn">Get in touch</button>
```

Comment:
- [ ] Kindly note that your `form` has to collect data by using the [Formspree service](https://formspree.io/html).
   -  `Form` uses the action and method attribute to integrate with [formspree service](https://formspree.io/html).
      * `action` value should be similar to `https://formspree.io/f/{form_id}`, you get the `id` from `Formspree`.
      * `method` value equals `POST`.

```suggestion
            </h2>
            <form class="form" action="https://formspree.io/f/{form_id}" method="post">
                <input class="form-list" type="text" placeholder="Full Name" required maxlength="30">
                <input class="form-list" type="email" placeholder="Email Address" required>
                <textarea class="form-textarea" placeholder="Write me something..." maxlength="500" required></textarea>    
                <button class="botn">Get in touch</button>
```